src/transformers/file_utils.py

```diff
@@ -300,11 +300,16 @@ def is_torch_cuda_available():
 _torch_fx_available = _torch_onnx_dict_inputs_support_available = False
 if _torch_available:
     torch_version = version.parse(importlib_metadata.version("torch"))
-    _torch_fx_available = _torch_onnx_dict_inputs_support_available = (torch_version.major, torch_version.minor) == (
+    _torch_fx_available = (torch_version.major, torch_version.minor) == (
         TORCH_FX_REQUIRED_VERSION.major,
         TORCH_FX_REQUIRED_VERSION.minor,
     )
 
+    _torch_onnx_dict_inputs_support_available = (torch_version.major, torch_version.minor) >= (
+        TORCH_ONNX_DICT_INPUTS_MINIMUM_VERSION.major,
+        TORCH_ONNX_DICT_INPUTS_MINIMUM_VERSION.minor
+    )
```

Comment:
```suggestion
    _torch_onnx_dict_inputs_support_available = torch_version >= TORCH_ONNX_DICT_INPUTS_MINIMUM_VERSION
```
You can directly compare the parsed version :-)